doc/STREAMING_ITERABLES.md

```diff
@@ -7,11 +7,144 @@
 - [Iterable Streams](#iterable-streams)
   - [Table of Contents](#table-of-contents)
   - [Usage Guide](#usage-guide)
+    - [Wrapping Duplex Iterables](#wrapping-duplex-iterables)
+  - [Iterable Stream Types](#iterable-stream-types)
+    - [Source](#source)
+    - [Sink](#sink)
+    - [Transform](#transform)
+    - [Duplex](#duplex)
   - [Iterable Modules](#iterable-modules)
 
 ## Usage Guide
 
-**Coming Soon!**
+### Wrapping Duplex Iterables
+
+Sometimes you may need to wrap an existing duplex stream in order to perform incoming and outgoing [transforms](#transform) on data. This type of wrapping is commonly used in encryption streams. Using [it-pair][it-pair] and [it-pipe], we can do this rather easily, given an existing [duplex iterable](#duplex).
+
+```js
+const duplexPair = require('it-pair/duplex')
+const pipe = require('it-pipe')
+
+// Wrapper is what we will write and read from
+// This gives us two duplex iterables that are internally connected
+const wrapper = duplexPair()
+
+// Now we can pipe our wrapper to the existing duplex iterable
+pipe(
+  wrapper[0], // Use one half of our pairs as the primary wrapper
+  outgoingTransform, // A transform iterable to send data through (ie: encrypting)
+  existingDuplex, // The original duplex iterable we are wrapping
+  decrypt, // A transform iterable to read data through (ie: decrypting)
+  wrapper[0]
+)
+
+// We can now read and write from the other half of our pair
+pipe(
+  ['some data'],
+  wrapper[1],
+  async (source) => {
+    for await (const chunk of source) {
+      // Chunk will be a BufferList, we can slice it to get the underlying buffer
+      console.log('Data: %s', chunk.slice())
```

Comment:
You could just call `toString()`. In the example there's nothing to say what `existingDuplex` is yielding so personally I wouldn't complicate it.

https://github.com/rvagg/bl#toString

```suggestion
      console.log('Data: %s', chunk.toString())
```